tmuxcolors-256.conf

```
@@ -1,27 +1,26 @@
 #### COLOUR (Solarized 256)
 
 # default statusbar colors
-set-option -g status-bg colour235 #base02
-set-option -g status-fg colour136 #yellow
-set-option -g status-attr default
+set-option -g status-style bg=colour235 #base02
```

The second command overrides the first, parameters must be inlined:
```suggestion
set-option -g status-style bg=colour235,fg=colour136 #base02,yellow
```